src/ONYXKEYS.js

```diff
@@ -171,5 +171,6 @@ export default {
     FORMS: {
         ADD_DEBIT_CARD_FORM: 'addDebitCardForm',
         REQUEST_CALL_FORM: 'requestCallForm',
+        VALIDATION_STEP_FORM: 'validationStepForm',
```

Comment:
I think we should use a single key for all VBBA form steps like we currently do, instead of a separate key for each step. We should use `REIMBURSEMENT_ACCOUNT` instead, but maybe we can move it to `FORMS.REIMBURSEMENT_ACCOUNT`

Reply:
I think this will require change in lots of files, should I create a separate PR for it?

Reply:
bump @luacmartins, I just noticed that the other external refactors also depend on this key, should I use the `REIMBURSEMENT_ACCOUNT` key that is outside of the `FORMS` key in this PR, until all of the other refactors that depend on it are merged, then we can create a separate PR to move the key under `FORMS`. Let me know what you think!

Reply:
Sounds good to me! Let's go with that.